source/2020-12-18-the-ember-times-issue-170.md

```diff
@@ -17,6 +17,20 @@ READMORE
 
 ---
 
+## [Ember 3.23 Released 🐹](https://blog.emberjs.com/2020/12/14/ember-3-23-released.html)
+
+Version 3.23 of Ember.js, Ember Data, and Ember CLI was released on December 14, 2020! The next release will be in [6 weeks time](http://emberjs.com/blog/2013/09/06/new-ember-release-process.html), please use the [ember-try](https://github.com/ember-cli/ember-try) addon to continuously test your projects against the latest Ember releases.
```

Comment:
Can we remove the mention of the 6 weeks time? Because 3.23 was released late, I'm not sure around when we can expect 3.24 release.

```suggestion
Version 3.23 of Ember.js, Ember Data, and Ember CLI was released on December 14, 2020! Please use the [ember-try](https://github.com/ember-cli/ember-try) addon to continuously test your projects against the latest Ember releases.
```